packages/atomic/src/components/common/result-link/attributes-slot.ts

```diff
@@ -5,16 +5,41 @@ export function getAttributesFromLinkSlot(host: HTMLElement, slotName: string) {
   if (!attributesSlot) {
     return;
   }
+  return extractAttributeFromLink(attributesSlot, slotName);
+}
+
+export function getLinkAttributesFromSlotChild(
```

Comment:
Unfortunately, Stencil does not recognize the HTMLSlotElement during tests so this method cannot be unit tested properly. I added a cypress test though
https://github.com/ionic-team/stencil/issues/2830